lib/fathead/django/fetch.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+mkdir -p download
+cd download
+rm -f *.html
+
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/contrib/ -O contrib.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/django-admin/ -O django-admin.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/exceptions/ -O exceptions.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/forms/fields/ -O form_fields.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/forms/widgets/ -O widgets.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/templates/builtins/ -O builtins.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/settings/ -O settings.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/validators/ -O validators.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/views/ -O view.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/urlresolvers/ -O urlresolvers.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/urls/ -O urls.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/models/database-functions/ -O database-functions.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/models/fields/ -O fields.html
+wget --quiet https://docs.djangoproject.com/en/1.10/ref/migration-operations/ -O migration-operations.html
```

Comment:
@vigzmv this is a great start -- the structure of the Django docs clearly doesn't lend itself to being parsed for classes/methods/attributes/etc

I have another idea though, what about downloading the [HTML offline docs](HTML) (instead of several files) and iterating over the [Index](https://docs.djangoproject.com/en/1.10/genindex/)?

It seems that most entries in the index indicate if they are a class, method, attribute or function in parenthesis. By scraping those and using the css classes on the page, I think we could reliably parse out all the methods, classes, attributes and functions from the docs?